main.go

```diff
@@ -265,9 +270,17 @@ func generate(in configInput) ([]byte, error) {
 		// the top-level server config. For more details, see:
 		// https://github.com/grpc/proposal/blob/master/A47-xds-federation.md#bootstrap-config-changes.
 		c.Authorities = map[string]Authority{
-			tdURI: {},
-			"":    {},
+			"": {},
 		}
+
+		if in.includeXDSTPNameInLDS {
+			c.Authorities[tdAuthority] = Authority{
+				ClientListenerResourceNameTemplate: fmt.Sprintf("xdstp://%s/envoy.config.listener.v3.Listener/%%s", tdAuthority),
+			}
+		} else {
+			c.Authorities[tdURI] = Authority{}
```

Comment:
We were saying we think this was a bug. Instead of s/tdURI/tdAuthority/ let's just remove this line entirely? Then the new flag being added in this PR adds the authority entry for tdAuthority.

Reply:
Done. 👍